framework/components/ecl-profile-topbars/ecl-profile-topbars.scss

```diff
@@ -0,0 +1,31 @@
+/**
+ * ECL Profile Topbar
+ * @define profile-topbar
+ */
+
+.ecl-profile-topbar {
+  margin: 0;
+}
+
+// This is esentially .section__group which can be used when refactored.
+.ecl-profile-topbar__wrapper {
+  margin-bottom: map-get($ecl-spacing, 'm');
+  margin-top: map-get($ecl-spacing, 'm');
+}
+
+.ecl-profile-topbar__pane {
+  height: 100%;
+  min-height: 160px;
+  position: relative;
+
+  > .ecl-profile-topbar__pane-button {
+    bottom: 0;
+    margin-bottom: 0;
+    position: absolute;
```

Comment:
Could have use a display flex instead of position absolute.
That works well here, so we can keep it, but if the component grow larger (with several context-navs), there will be overlaps.